### PR TITLE
Update Dockerfile

### LINKED ISSUE #363: Dockerize the scenario player
Analysis
=======
Current State
-----------------
We currently pull the raiden and SP repo grom github via a bash script, build a docker container and use it in the nightlyies, every night.

Problem
-----------
The bash script is delicate and not exactly robust, the process of building a docker container is
additional load which can be easily offloaded to dockerhub.

Solution
----------
Write a new docker setup for the scenario player

### PR DESCRIPTION
**Pull request by nlsdfnbch**
Fixes #363

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,62 @@
...
+#
+# Copies the raiden client, scenario player and scenario defnition files from the
+# compilation stage, and configure further environment variables required to run
+# the nightlies.
+#
+# This stage copies the virtual environment created in the previous stage,
+# and prepends its location to the $PATH environment variable.
+
+# Copy virtual env and scenario definition files.
+COPY --from=compile-image /opt/venv /opt/venv
```
Comment (konradkonrad) [inline Dockerfile]: Depending on how you intend to use this, it would make sense to copy the scenarios from the raiden checkout above.

Comment (konradkonrad) [reply Dockerfile]: The alternative is of course to mount the scenario descriptions into the running container (which will need another git checkout step previous to the execution)

**Comment by nlsdfnbch** [comment]
Scenarios are now part of the container

File: Dockerfile (hunk 1 above)
Comment (nlsdfnbch) [reply Dockerfile]: Scenarios are now part of the container

**Comment by nlsdfnbch** [comment]
Yepp, unrelated

**Pull request merged**